examples/ws/rest/README.md

```diff
@@ -0,0 +1,55 @@
+# README
+
+## REST examples
+
+### JavaScriptForRest
+
+Follow these steps.
+
+`cd ~/JavaScriptForRest`\
+`php -S localhost:8000`
+
+In your default browser goto `http://localhost/8000/popcornA.html`\
+or `http://localhost/8000/popcornA.html`\
+or `http://localhost/8000/objectInheritance_Person1.html`
+
+### MashupTest
+
+This project demonstrates mashup from different services and converging the results into one.<br/>
+If logging in remotely from laptop or home pc<br/>
+
+`ssh -X uname_encs@computation.encs.concordia.ca`<br/>
+
+else<br/>
+
+Login to a encs lab pc and<br/>
+`openesb --userdir /groups/alphabet/gname_487_x/temp/&`<br/>
+
+In the services tab add a new server instance e.g. `GlassFish_Server`.<br/>
+It can be downloaded on the go from within the IDE into your `/group-directory` or the `/nettemp-directory` .<br/>
+
+For Apache Tomcat in the ENCS environment you need to use the wrapper command `tomcat7_setup /path/to/folder/that/will/become/CATALINE_BASE`<br/>
+
+For example:<br/>
+`[orwell][/groups/x/xyz_soen487_i] > tomcat7_setup xyzTomcat`<br/>
+The output of the above command should look similar to the following,<br/>
+
+`Subdirectory xyzTomcat created`
```

Comment:
A big bulk of text can be quoted using triple ``` marks at the beginning and the end.

Reply:
@jashanj0tsingh -- on the 2nd though all the ENCS-stuff should be in your tutorial instead, not in here. Only keep generic stuff.